rpc/interface/src/route/fallback.rs

```diff
@@ -0,0 +1,18 @@
+//! Fallback route functions.
+
+//---------------------------------------------------------------------------------------------------- Import
+use axum::http::StatusCode;
+
+//---------------------------------------------------------------------------------------------------- Routes
+/// Fallback route function.
+///
+/// This is used as the fallback endpoint in [`crate::RouterBuilder`].
+pub(crate) async fn fallback() -> StatusCode {
+    StatusCode::NOT_FOUND
+}
```

Comment:
Fallback route function, i.e. the behavior to do on `/unknown_endpoints`. This currently follows `monerod`, it replies with a 404 with no body.

We can change this later.